pybikes/metro_bike_share.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+class MetroBikeShare(BikeShareSystem):
+
+    sync = True
+
+    meta = {
+        'system': 'Metro Bike Share',
+        'company': ['Los Angeles County Metropolitan Transportation Authority (Metro)', 'Bicycle Transit Systems Inc', 'B-cycle'],
+    }
+
+    def __init__(self, tag, feed_url, meta):
+        super(MetroBikeShare, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+            scraper.ssl_verification = False
```

Comment:
Ok, thank you for the answer. I will try to upgrade my python version to 2.7.9. and see if the error disappears.